docs/modules/ROOT/pages/explanations/decisions/mariadb.adoc

```diff
@@ -109,18 +114,27 @@ Advantages::
 * Sufficient Galera support
 * Well tested and widely used by VSHN
 * Faster integration and development as we already know how to work with it
+* Potentially easier to move the SPKS project to AppCat
 
 Disadvantages::
 
 * Lack of regular, automated maintenance
-* We need to use other tools for user/logical database management.
+* No user or user/logical database management.
+* Neither Backup nor automatic restore
 
 == Decision
 
 We use the Bitnami Helm Chart.
 
 == Rationale
 
-The mariadb-operator project seems to be a very good solution with high potential, but it is not yet mature enough for us to use it. We would need to invest much more time to integrate it into our offerings and maintain it. It's developed by one maintainer only, which is risky for us.
+The mariadb-operator project seems to be a very good solution with high potential, but it is not yet mature enough for us to use it. We would need to invest much more time to integrate it into our offerings and maintain it. It's developed by one maintainer only, which is somewhat risky for us.
```

Comment:
Afaik it is mature enough for us. The issue is more with the risk of being abandoned